package ExceptionHandling;

public class Main 
{
	public static void main(String[] args) 
	{
		int c = 0;
		
		try
		{
			c = 4/0;
		}
		catch(ArithmeticException e)
		{
			System.out.println("you broke it");
		}
		catch(Exception e)
		{
			System.out.println("you really broke it");
		}
	}
}
